src/components/search/Filter/Filter.tsx

```diff
@@ -202,34 +80,41 @@ function Filter({
               aria-label="Close Filters"
               icon={<Icon name="X" width={32} height={32} />}
               onClick={() => {
-                setSelectedFacets(searchState.selectedFacets)
+                dispatch({
+                  type: 'selectFacets',
+                  payload: selectedFacets,
+                })
+
                 dismissTransition.current?.()
               }}
             />
           </header>
           <Facets
+            facets={facets}
```

Comment:
There's a way to reuse the same Facets component instead of rendering two?

Reply:
Not really, because the callbacks are different